### PR TITLE
Update the update instructions with per-method details

### LINKED ISSUE #7881: Improve updating docs (UPDATE.md) with per-method instructions
During the big overhaul of installation docs (#7841), @Ferroin pointed out some issues with the document about updating Netdata. The product team has recently informed me that we'll need a very concise and up-to-date document about updating Netdata agents to help users access Netdata Cloud once the current entrypoint (Nodes View) is disabled.

This document should contain specific update methods for the following methods:

- kickstart.sh
- kickstart-static64.sh
- Manual installation
- Packages
- Docker (?)

Austin had a few specific comments, which I've added below.

```
Netdata versions older than `v1.12.0-rc2-52` had a `netdata-updater.sh` script in the root directory of the source code,
which has now been deprecated. The manual process that works for all versions to get the latest commit in git is to use
the `netdata-installer.sh`. The installer preserves your custom configuration and updates the information of the
installation in the `.environment` file under the user configuration directory.
```

> Unless I'm mistaken, kickstart.sh works too, and is probably a safer update method as it will ensure that any new dependencies get pulled in. It also preserves user configuration just like netdata-installer.sh (it actually calls out to netdata-installer.sh to do the actual install).

```
If a newer version of Netdata is not available from the source you got it, we suggest to uninstall the version you have
and follow the [installation](README.md) instructions for installing a fresh version of Netdata.
```

> They should never need to go through this, they should be going through the regular package manager based workflow for their system to update the package.

### PR DESCRIPTION
**Pull request by joelhans**
##### Summary

Fixes #7881 

To support the upcoming Cloud release, we need to have better instructions to help users update their agents to the latest version and access the new features. If their agent is old and they try to access deprecated features, they'll be directed to this document.

I got rid of most of the existing document and replaced it with instructions specific to different methods of installing Netdata.

Putting this up as a WIP so we can discuss the broad strokes and figure out what major pieces might be missing from what I've whipped up today. I'm not sure how deeply we want to (or can) dive into various OS/distros/versions.

cc @manos-saratsis 

##### Component Name

docs
packaging/installer

### CODE REVIEW

File: packaging/installer/UPDATE.md [hunk 1]
```diff
@@ -1,74 +1,158 @@
...
 
-```sh
-# go to the git downloaded directory
-cd /path/to/git/downloaded/netdata
+If you installed Netdata using our one-line automatic installation script, run it again to update Netdata. Any custom
+settings present in your Netdata configuration directory (typically at `/etc/netdata`) will persist during this process.
 
-# update your local copy
-git pull
+This script will download the latest Netdata source (either the nightly or stable version), compile Netdata, and update
```
Comment (shortpatti) [inline packaging/installer/UPDATE.md]:
```suggestion
This script downloads the latest Netdata source (either the nightly or stable version), compile Netdata, and update
```

File: packaging/installer/UPDATE.md [hunk 2]
```diff
@@ -1,74 +1,158 @@
...
-The `kickstart.sh` and `kickstart-static64.sh` one-liners will do a one-time update to the latest nightly build, if
-executed as follows:
+If you installed Netdata using the pre-built static binary, run the `kickstart-static64.sh` script again to update
+Netdata. Any custom settings present in your Netdata configuration directory (typically at `/etc/netdata`) will persist
+during this process.
 
-```sh
-# kickstart.sh
-bash <(curl -Ss https://my-netdata.io/kickstart.sh) --no-updates
+This script will download the latest Netdata source (either the nightly or stable version), compile Netdata, and update
```
Comment (shortpatti) [inline packaging/installer/UPDATE.md]:
```suggestion
This script downloads the latest Netdata source (either the nightly or stable version), compile Netdata, and update
```

File: packaging/installer/UPDATE.md [hunk 3]
```diff
@@ -1,74 +1,158 @@
...
-first place.
+## macOS
+
+If you installed Netdata on your macOS system using Homebrew, you can explictly request an update:
+
+```bash
+brew upgrade netdata
+```
+
+Homebrew will download the latest Netdata via the
```
Comment (shortpatti) [inline packaging/installer/UPDATE.md]:
```suggestion
Homebrew downloads the latest Netdata via the
```

File: packaging/installer/UPDATE.md [hunk 4]
```diff
@@ -1,74 +1,158 @@
...
+If you installed Netdata manually from Git using `netdata-installer.sh`, you can run that installer again to update your
+agent. First, run our automatic requirements installer, which works on many Linux distributions, to ensure your system
+has the dependencies necessary for new features.
+
+```bash
+bash <(curl -sSL https://raw.githubusercontent.com/netdata/netdata/master/packaging/installer/install-required-packages.sh)
+```
+
+Then, navigate to the directory where you first cloned the Netdata repository, pull the latest source code, and run
+`netdata-install.sh` again. This process will compile Netdata with the latest source code and update it via
```
Comment (shortpatti) [inline packaging/installer/UPDATE.md]:
```suggestion
`netdata-install.sh` again. This process compiles Netdata with the latest source code and update it via
```

Comment (joelhans) [reply packaging/installer/UPDATE.md]: Fixed all these! Plus a few others. I was very into the future when I wrote this, apparently.

**Comment by manos-saratsis** [comment]
@joelhans see https://github.com/netdata/product/issues/14 for the new update icon. Design is [here](https://www.figma.com/file/XqJc6aK9WDZ0dxq3Xh61BN/%E2%9C%85-S1.0-Navbars-Cloud-and-Agent?node-id=747%3A12130)

**Comment by joelhans** [comment]
@Ferroin && @prologic: Any thoughts here? There's not an immediate rush, but I would like to get this merged before the end of the week in case we need to make last-minute changes ahead of the release.

**Pull request merged**